actions/asana-create-pr-subtask/asana-create-pr-subtask.sh

```diff
@@ -58,8 +59,8 @@ _check_pr_subtask_exist() {
 		parent_name=$(jq -r '.parent_name' <<< "$item" | awk '{$1=$1};1')
 		assignee=$(jq -r '.assignee' <<< "$item")
 
-		if [[ "$task_name" == "${pr_prefix}"* && "$task_name" == *"$parent_name"* && "$assignee" == "$asana_assignee_id" ]]; then
-			echo "$item"
+		if [[ "$task_name" == "${pr_prefix} ${parent_name} (${github_repo_name})" && "$assignee" == "$asana_assignee_id" ]]; then
```

Comment:
Maybe be less strict about the name? Just assert the prefix and the suffix:

```suggestion
		if [[ "$task_name" == "${pr_prefix}*(${github_repo_name})" && "$assignee" == "$asana_assignee_id" ]]; then
```

Reply:
I’ve changed it according to your suggestion